modoboa/lib/templatetags/lib_tags.py

```diff
@@ -94,3 +95,12 @@ def extra_static_content(caller, st_type, user):
                 "GetStaticContent", caller, st_type, user)
         })
     )
+
+
+@register.filter(name='localize_header_name')
+def localize_header_name(headername):
+    """ Localizes the header names """
+    names = {'From': _('From'), 'To': _('To'), 'Date': _('Date'), 'Subject': _('Subject')}
```

Comment:
Could you please use " instead of ' for strings. I'm currently updating the code to be consistent but it's a long path...